modules/apps/address/address-impl/src/main/resources/com/liferay/address/dependencies/regions/IL.json

```diff
@@ -1,26 +1,26 @@
 [
 	{
-		"name": "HaDarom",
+		"name": "Al Awsaţ",
```

Comment:
@wanderlast should the denomination of the regions should be in the local language of the country in Hebrew in this case instead of in Arab? Thanks